pkg/products/marin3r/reconciler.go

```diff
@@ -229,7 +230,13 @@ func (r *Reconciler) Reconcile(ctx context.Context, installation *integreatlyv1a
 }
 
 func (r *Reconciler) reconcileAlerts(ctx context.Context, client k8sclient.Client, installation *integreatlyv1alpha1.RHMI) (integreatlyv1alpha1.StatusPhase, error) {
-	alertReconciler, err := r.newAlertsReconciler()
+
+	granafaConsoleURL, err := grafana.GetGrafanaConsoleURL(ctx, client, installation)
+	if err != nil {
+		logrus.Errorf("failed to get Grafana console URL %w", err)
```

Comment:
Should we return a phaseawaitingcomponents here rather than just log? There could be a case where the url is `""`? Or are we okay to pass an empty string to the newAlertsReconciler until it's found? A case where we don't want to stop the reconcile perhaps.

Reply:
Yeah, I didn't want to stop the reconcile just because there was an issue getting the Granafa URL, it could prevent the operator from completing an upgrade for example.

Reply:
Cool cool cool. I'm fine with that. We do it elsewhere. 👍